metron-analytics/metron-statistics/src/test/java/org/apache/metron/statistics/StellarStatisticsFunctionsTest.java

```diff
@@ -356,6 +357,49 @@ public void testSkewness() throws Exception {
     assertEquals(stats.getSkewness(), (Double) actual, 0.1);
   }
 
+  @Test
+  public void testStatsBin() throws Exception {
```

Comment:
It's not immediately obvious to me what's being tested here - is this checking that default enums' list of splits is syntactically sound? What would make this test fail?